tests/integration/test_kube_orchestrator.py

```diff
@@ -84,12 +85,10 @@ def _prepare(self) -> None:
         assert isinstance(self._orchestrator, KubeOrchestrator)
         namespace = self._orchestrator.kube_config.namespace
 
-        self.internal_hostname = f"{self.id}.{namespace}"
+        self._record.internal_hostname = f"{self.id}.{namespace}"
         if self.is_named:
-            from platform_api.handlers.validators import JOB_USER_NAMES_SEPARATOR
-
-            self.internal_hostname_named = (
-                f"{self.name}{JOB_USER_NAMES_SEPARATOR}{self.project_name}.{namespace}"
+            self._record.internal_hostname_named = (
+                f"{self.name}{JOB_NAME_SEPARATOR}{self.project_name}.{namespace}"
```

Comment:
minor: seems to be slightly different from actual implementation, but it influences only tests

Reply:
yes, I decided to leave it since it doesn't matter how hostname looks like in tests